web/src/components/dashboard/SidebarCardsContainer.test.tsx

```diff
@@ -77,6 +77,40 @@ describe("<SidebarCardsContainer />", () => {
   };
 
   describe("nudges", () => {
+    it("renders a card in the top section if set to above-opportunities", () => {
+      const sidebarCards = {
+        "funding-nudge": generateSidebarCardContent({
+          id: "funding-nudge",
```

Comment:
Maybe we should invent an ID instead of using an existing ID, so we don't run into the same confusion again in the future regarding whether this test is testing the "funding-nudge"?

Reply:
I just called it fake-above-card and fake-below-card... Always up for opinions on naming